python/lsst/rubintv/production/allSky.py

```diff
@@ -412,7 +414,12 @@ def animateFilesAndUpload(self, isFinal=True):
                 raise RuntimeError(f'Failed to find movie {creationFilename}')
 
         if not self.DRY_RUN:
-            self.uploader.googleUpload(self.channel, creationFilename, uploadAsFilename)
+            self.uploader.googleUpload(self.channel, creationFilename, uploadAsFilename, isLargeFile=True)
```

Comment:
Is adding the `isLargeFile=True` part an independent change from adding epoUploader?

Reply:
It's just a bugfix really. I realised I should be using it for the EPO uploader when I wrote the line, and spotted that I didn't have it set for the normal upload, so just set it there too. It'll go away soon enough though, as all the normal "upload" stuff will be S3 instead of GCS soon.